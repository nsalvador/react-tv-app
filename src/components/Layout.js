import React from 'react';

import Header from './Header';
import Footer from './Footer';

const Layout = ({ children }) => (
	<div className="layout-container">
		<div className="layout-content">
			<Header />
			{children}
		</div>
		<Footer />
	</div>
);

export default Layout;
