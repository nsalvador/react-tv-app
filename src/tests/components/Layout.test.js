import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Layout from '../../components/Layout';

it('should render Layout correctly', () => {
	const renderer = new ReactShallowRenderer();
	renderer.render(<Layout />);
	expect(renderer.getRenderOutput()).toMatchSnapshot();
});
