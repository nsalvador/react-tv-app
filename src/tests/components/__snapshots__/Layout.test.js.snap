// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Layout correctly 1`] = `
<div>
  <Header />
  <Footer />
</div>
`;
